boards/px4/fmu-v5/default.px4board

```diff
@@ -78,6 +78,9 @@ CONFIG_MODULES_RC_UPDATE=y
 CONFIG_MODULES_ROVER_POS_CONTROL=y
 CONFIG_MODULES_SENSORS=y
 CONFIG_MODULES_SIMULATION_PWM_OUT_SIM=y
+CONFIG_MODULES_SIMULATION_SENSOR_BARO_SIM=y
```

Comment:
We could make this kconfig dependencies for CONFIG_MODULES_SIMULATION_SIMULATOR_SIH.